### DISCUSSION TITLE
plugin hook message is dropped

### DISCUSSION THREAD

**Discussion by itweb62** [halfgaar/FlashMQ#46] [General]
Hello,

I want to know if it's possible for DB persistence.

**Comment by bigsmoke** [comment]
Hi @itweb62, You want to know if _what_ is possible? Your message contains no details beyond the subject “plugin hook message is dropped”… Am I missing something?

**Reply by itweb62** [reply]
Soory i have made a misstake it's good for me

**Comment by halfgaar** [comment]
Alright, sounds good. I'll close the discussion.
